bottlerocket/samples/RUNBOOK.md

```diff
@@ -1,438 +1,57 @@
 # Bottlerocket Test System Sample Files
 
-These files contain templates that can be populated using environment variables and the `eval` command and run with the `cli` tool.
+These files contain templates that can be populated and run using `cargo make` commands.
 
-Examples of how to populate each one of these files can be found below.
+Examples of how to populate and run each one of these files can be found below.
 
-Each codeblock starts with a small number of variables that need to be populated before running the block, followed by a number of variables that contain default values (but can be modified by the user).
+Before running a `cargo make` command, several environment variables can be set to customize the generated file.
 
-_Note_: `ASSUME_ROLE` has a default value of `~` (null), but you can replace this with the ARN of an AWS IAM role that should be used for all AWS calls.
-
-The final `cat` command will print the populated file to the path indicated by `OUTPUT_FILE`.
-
-## EKS
+The directory from which to use the sample file corresponds with the value in `CLUSTER_TYPE`.
 
 The files in [eks](./eks) are meant to be run on an EKS test cluster. You can create a new cluster using the [eksctl](https://eksctl.io/introduction/) tool.
```

Comment:
I don't think we've used the term "management cluster" before, but maybe we should, to distinguish it from the target clusters.

```suggestion
The files in [eks](./eks) are meant to be run on an EKS management cluster. You can create a new cluster using the [eksctl](https://eksctl.io/introduction/) tool.
```